### DISCUSSION TITLE
Google Maps version 6?

### DISCUSSION THREAD

**Discussion by gm112** [ionic-team/capacitor#7438] [Q&A] [answered]
I haven't seen any communication regarding the Google Maps plugin and was wondering if a 6.0 release is planned? The repo appears to have moved, and I do see that there is a release branch, but the latest package is still locked on version 5 for capacitor.

Just wanting to inquire because I'm curious and haven't had luck finding answers is all!

**Comment by jcesarmobile** [comment, marked as answer]
see https://github.com/ionic-team/capacitor-google-maps/issues/5#issuecomment-2072345149

**Reply by gm112** [reply]
Thank you so very much!

**Reply by tharrington** [reply]
This still didn't work for me on android. Are there more specific instructions to upgrade?

**Reply by nikola418** [reply]
> This still didn't work for me on android. Are there more specific instructions to upgrade?

Same here :(((

**Reply by jcesarmobile** [reply]
The final version 6 of the plugin was released a few weeks ago, you don't need to install the `next` version but `latest`.

If you find issues with latest version, report them on https://github.com/ionic-team/capacitor-google-maps/issues
